testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -822,7 +822,7 @@ public void getRowsWithQuery() throws IOException {
     JsonNode data = getResponseWrapper.getData();
     assertThat(data.size()).isEqualTo(1);
     assertThat(data.at("/0/id").asText()).isEqualTo(rowIdentifier);
-    assertThat(data.at("/0/firstName").asText()).isEqualTo("John");
+    assertThat(data.at("/0/firstname").asText()).isEqualTo("John");
```

Comment:
This is due to unfortunate discrepancy between SGv1 and SGv2 for this test class; not a difference in behavior per se (that is, SGv2 creates field with lower-case name so it is returned as such -- handling itself is case-insensitive).